content/docs/guides/certificates.md

```diff
@@ -37,6 +37,35 @@ data:
 
 For details and code where this is used see [osm-controller.go](https://github.com/openservicemesh/osm/blob/release-v0.9/cmd/osm-controller/osm-controller.go#L182-L183).
 
+#### Rotate the Root Certificate
+
+The self-assigned root certificate, which is created via the Tresor package within OSM, will expire in a decade. To rotate the root cert, the following steps should be followed: 
```

Comment:
```suggestion
The self-signed root certificate, which is created via the Tresor package within OSM, will expire in a decade. To rotate the root cert, the following steps should be followed: 
```

Reply:
Root certificate rotation applies to certificate providers other than Tresor as well. Could you clarify in the documentation whether the process to rotate the root cert is the same for all certificate providers, and additionally the steps for each of them if different? I think for Vault we do not create the osm-ca-bundle secret.